client/src/Pages/artists.jsx

```diff
@@ -12,7 +12,8 @@ function Artists() {
   const [artists, setArtists] = React.useState([]);
   const changeState = (prop) => { setArtists(prop); };
   if (artists.length === 0) {
-    axios.get('http://localhost:3001/api/artist').then((res) => { const val = res.data; changeState(val); });
+    console.log(`Process.env.REACT_APP_BASE_URL: ${process.env.REACT_APP_BASE_URL}`);
```

Comment:
Why do we have a console.log here ? When I compile with docker this create an error